Mathlib/Order/Basic.lean

```diff
@@ -955,21 +955,76 @@ theorem const_lt_const : const β a < const β b ↔ a < b := by simpa [Pi.lt_de
 
 end Function
 
-/-! ### Lifts of order instances -/
+/-! ### Pullbacks of order instances -/
+
+/-- Pull back a `Preorder` instance along an injective function.
+
+See note [reducible non-instances]. -/
+abbrev Function.Injective.preorder [Preorder β] [LE α] [LT α] (f : α → β)
+    (le : ∀ {x y}, f x ≤ f y ↔ x ≤ y) (lt : ∀ {x y}, f x < f y ↔ x < y) :
+    Preorder α where
+  le_refl _ := le.1 <| le_refl _
```

Comment:
```suggestion
  le_refl _ := le.1 le_rfl
```

Reply:
This was a deliberate choice to match the field name